Tutorial.md

```diff
@@ -1,7 +1,8 @@
 # API Blueprint Tutorial
-Welcome to the API Blueprint Tutorial! In this tutorial you will learn all about all the key features of the API Blueprint Language on the example of an API for our imaginary industry-changing Gist service.
 
-Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set some expectations let's provide following functionality in our API: *List Gists, Retrieve a Single Gist, Create a Gist, Edit a Gist, Star a Gist, Unstar a Gist, Check if a Gist is starred and Delete a Gist.*
+Welcome to an API Blueprint Tutorial! In this tutorial you will learn about the basics of the API Blueprint language.
+
+In this tutorial, we’re going to build an API Blueprint for a service called Polls – a simple web service that allows consumers to view polls and vote in them.
```

Comment:
Do you want to refer to this as a Polls API rather than a service called Polls? Do you want to call this a "web service?"

I would replace "that" with something as well. "A simple web service for viewing polls and casting votes." Not saying you should use this example, but "that" is usually too general of a word for me. This may be mostly preference, too.